analytics_extension.go

```diff
@@ -0,0 +1,54 @@
+package helix
+
+// ExtensionAnalytic ...
+type ExtensionAnalytic struct {
+	ExtensionID string    `json:"extension_id"`
+	URL         string    `json:"URL"`
+	Type        string    `json:"type"`
+	dateRange   DateRange `json:"date_range"`
+}
+
+// DateRange ...
+type DateRange struct {
+	StartedAt string `json:"started_at"`
+	EndedAt   string `json:"ended_at"`
+}
+
+// ManyExtensionAnalytics ...
+type ManyExtensionAnalytics struct {
+	ExtensionAnalytics []ExtensionAnalytic `json:"data"`
+}
+
+// ExtensionAnalyticsResponse ...
+type ExtensionAnalyticsResponse struct {
+	ResponseCommon
+	Data ManyExtensionAnalytics
+}
+
+type ExtensionAnalyticsParams struct {
+	ExtensionID string `query:"extension_id"`
+	First       int    `query:"first"`
+	After       string `query:"after"`
+	StartedAt   string `query:"started_at"`
+	EndedAt     string `query:"ended_at"`
```

Comment:
Both `StartedAt` and `EndedAt` should be of type [`Time`](https://github.com/nicklaw5/helix/blob/master/time.go#L17).